src/trials.jl

```diff
@@ -338,115 +346,116 @@ Base.show(io::IO, t::TrialEstimate) = _show(io, t)
     else
-        print(io, pad, " No results.")
-        return
+        println(io)
+    end
+    if !all(iszero, gctimes)
+        printstyled(io, pad, "│  "; color=padcolor)
+        print(io, "GC time: mean ", avggctime)
+        printstyled(io, " (", avegcpercent, ")"; color=:green)
+        println(io, ", max ", maxgctime, " (", maxgcpercent, ")")
     end
-
-    lmaxtimewidth = maximum(length.((medtime, avgtime, mintime)))
-    rmaxtimewidth = maximum(length.((stdtime, maxtime)))
-    lmaxgcwidth = maximum(length.((medgc, avggc, mingc)))
-    rmaxgcwidth = maximum(length.((stdgc, maxgc)))
-
-    # Main stats
-
-    print(io, pad, " Range ")
-    printstyled(io, "("; color=:light_black)
-    printstyled(io, "min"; color=:cyan, bold=true)
-    print(io, " … ")
-    printstyled(io, "max"; color=:magenta)
-    printstyled(io, "):  "; color=:light_black)
-    printstyled(io, lpad(mintime, lmaxtimewidth); color=:cyan, bold=true)
-    print(io, " … ")
-    printstyled(io, lpad(maxtime, rmaxtimewidth); color=:magenta)
-    print(io, "  ")
-    printstyled(io, "┊"; color=:light_black)
-    print(io, " GC ")
-    printstyled(io, "("; color=:light_black)
-    print(io, "min … max")
-    printstyled(io, "): "; color=:light_black)
-    print(io, lpad(mingc, lmaxgcwidth), " … ", lpad(maxgc, rmaxgcwidth))
-
-    print(io, "\n", pad, " Time  ")
-    printstyled(io, "("; color=:light_black)
-    printstyled(io, "median"; color=:blue, bold=true)
-    printstyled(io, "):     "; color=:light_black)
-    printstyled(io, lpad(medtime, lmaxtimewidth), rpad(" ", rmaxtimewidth + 5); color=:blue, bold=true)
-    printstyled(io, "┊"; color=:light_black)
-    print(io, " GC ")
-    printstyled(io, "("; color=:light_black)
-    print(io, "median")
-    printstyled(io, "):    "; color=:light_black)
-    print(io, lpad(medgc, lmaxgcwidth))
-
-    print(io, "\n", pad, " Time  ")
-    printstyled(io, "("; color=:light_black)
-    printstyled(io, "mean"; color=:green, bold=true)
-    print(io, " ± ")
-    printstyled(io, "σ"; color=:green)
-    printstyled(io, "):   "; color=:light_black)
-    printstyled(io, lpad(avgtime, lmaxtimewidth); color=:green, bold=true)
-    print(io, " ± ")
-    printstyled(io, lpad(stdtime, rmaxtimewidth); color=:green)
-    print(io, "  ")
-    printstyled(io, "┊"; color=:light_black)
-    print(io, " GC ")
-    printstyled(io, "("; color=:light_black)
-    print(io, "mean ± σ")
-    printstyled(io, "):  "; color=:light_black)
-    print(io, lpad(avggc, lmaxgcwidth), " ± ", lpad(stdgc, rmaxgcwidth))
 
     # Histogram
 
-    histquantile = 0.99
-    # The height and width of the printed histogram in characters.
+    # Axis ends at this quantile, same as displayed time, ideally:
+    histquantile = showpercentile/100
+    # The height and width of the printed histogram in characters:
     histheight = 2
-    histwidth = 42 + lmaxtimewidth + rmaxtimewidth
+    histwidth = 74  # fits into 78 chars, as does 1st line with 4 times
```

Comment:
Is there any point checking `get(io, :displaysize, (24, 80))`? What if someone makes their terminal narrower than 80 characters?

Reply:
Oh yes, this is probably a great idea. I had a todo but I forgot.

The other thing I want to do for narrow displays (e.g. copying to slack!) is to delete all trailing spaces, since these cause extra wrapping.